internal/command/output_test.go

```diff
@@ -325,3 +325,71 @@ func TestOutput_stateDefault(t *testing.T) {
 		t.Fatalf("bad: %#v", actual)
 	}
 }
+
+func TestOutput_showSensitiveArg(t *testing.T) {
+	originalState := testStateForOutput()
+
+	statePath := testStateFile(t, originalState)
+
+	view, done := testView(t)
+	c := &OutputCommand{
+		Meta: Meta{
+			testingOverrides: metaOverridesForProvider(testProvider()),
+			View:             view,
+		},
+	}
+
+	args := []string{
+		"-state", statePath,
+		"-show-sensitive",
+	}
+	code := c.Run(args)
+	output := done(t)
+	if code != 0 {
+		t.Fatalf("bad: \n%s", output.Stderr())
+	}
+
+	actual := strings.TrimSpace(output.Stdout())
+	if actual != "foo = \"bar\"" {
+		t.Fatalf("bad: %#v", actual)
+	}
+}
+
+func TestOutput_withoutShowSensitiveArg(t *testing.T) {
+	originalState := testStateForOutput()
+
+	statePath := testStateFile(t, originalState)
+
+	view, done := testView(t)
+	c := &OutputCommand{
+		Meta: Meta{
+			testingOverrides: metaOverridesForProvider(testProvider()),
+			View:             view,
+		},
+	}
+
+	args := []string{
+		"-state", statePath,
+	}
+	code := c.Run(args)
+	output := done(t)
+	if code != 0 {
+		t.Fatalf("bad: \n%s", output.Stderr())
+	}
+
+	actual := strings.TrimSpace(output.Stdout())
+	if actual != "foo = <sensitive>" {
+		t.Fatalf("bad: %#v", actual)
+	}
+}
+
+func testStateForOutput() *states.State {
```

Comment:
Without context, this function is unclear. One would probably assume they could use this as the state for all tests in this suite

The name of it should be more precise, probably something like `stateWithSensitiveValue`

Reply:
makes sense. I have renamed the function to `stateWithSensitiveValueForOutput` to ensure uniqueness across test files in the same package. We could name it as `TestOutput_stateWithSensitiveValue`, but it might be misleading since this is just a helper function not a test case.